### PR TITLE
Adds Einstein Merc Equipment (Again)

### PR DESCRIPTION
**Pull request by BurpleBineapple**
### Overview
(New branch, because it would have taken longer to unravel the mess I made than to just make a new branch and paste the files there)

Adds two gear crates for mercenary teams (or an individual) looking to play as Einstein Engines.

The hardsuit is a carbon copy stat-wise of all other merc hardsuits, but contains an ion gun and more anti-synthetic equipment than hard lethals. The sprite is the same as the Einstein asset protection suit.

The group gear crate comes with:

x5 Einstein Voidsuits
x1 Einstein Hardsuit
x2 Energy Rifles
x1 Energy Shotgun
x4 9mm Pistols and Silencers
x8 Spare 9mm Magazines

The single gear crate comes with:

x1 Einstein Voidsuit
x1 Energy Rifle
x1 9mm Pistol and Silencer
x2 Spare 9mm Magazines

image
Picture of the "Paragon" hardsuit.

### Asset Licenses
The Paragon hardsuit sprites were made by Kyres and provided by him during a chat in the code_dungeon channel. Thanks!

### CODE REVIEW

**Comment by BurpleBineapple** [comment]
!review

File: code/modules/clothing/spacesuits/rig/suits/merc.dm [hunk 1]
```diff
@@ -115,4 +115,25 @@
...
+	initial_modules = list(
+		/obj/item/rig_module/power_sink,
+		/obj/item/rig_module/mounted/ion,
+		/obj/item/rig_module/cooling_unit,
+		/obj/item/rig_module/actuators/combat,
+		/obj/item/rig_module/maneuvering_jets,
+		/obj/item/rig_module/ai_container,
+		/obj/item/rig_module/vision/multi,
+		/obj/item/rig_module/electrowarfare_suite
+		)
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/rig/suits/merc.dm:139]:
```suggestion
	)
```

File: html/changelogs/purplepineapple-PR-einstein.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+author: PurplePineapple, Kyres
+
+delete-after: True
+
+changes:
+  - rscadd: "Added a variant of the mercenary hardsuit with a sprite made by Kyres for the Einstein group gear crate."
```
Comment (SleepyGemmy) [inline html/changelogs/purplepineapple-PR-einstein.yml:6]:
```suggestion
  - rscadd: "Added a variant of the mercenary hardsuit for use in the Einstein gear crates."
```

File: code/modules/clothing/spacesuits/rig/suits/merc.dm (hunk 1 above)
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/rig/suits/merc.dm:139]:
```suggestion
	)
```

**Review by SleepyGemmy — APPROVED** [review]

File: code/modules/clothing/spacesuits/rig/suits/merc.dm [hunk 3]
```diff
@@ -115,4 +115,25 @@
...
+	suit_type = "paragon"
+	icon = 'icons/clothing/rig/einstein_merc.dmi'
+	icon_state = "paragon"
+	icon_supported_species_tags = null
+	allowed_module_types = MODULE_GENERAL | MODULE_LIGHT_COMBAT | MODULE_HEAVY_COMBAT | MODULE_SPECIAL | MODULE_MEDICAL | MODULE_UTILITY
+	species_restricted = list(BODYTYPE_HUMAN, BODYTYPE_IPC, BODYTYPE_IPC_BISHOP, BODYTYPE_IPC_ZENGHU)
+
+	initial_modules = list(
+		/obj/item/rig_module/power_sink,
+		/obj/item/rig_module/mounted/ion,
```
Comment (Arrow768) [inline code/modules/clothing/spacesuits/rig/suits/merc.dm:132]: I am a bit unsure about the mounted ion rifle, as ion weapons are not easily available during standard gameplay and emps arnt exactly balanced atm.

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**